index.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         var dotFolders            = this.readConfig('dotFolders');
         var serverSideEncryption  = this.readConfig('serverSideEncryption');
         var batchSize             = this.readConfig('batchSize');
+        var defaultMimeType       = this.readConfig('defaultMimeType');
```

Comment:
@wytlytningNZ Yep, exactly. That's what the README says it is. So, what I meant was, would you mind adding that default in the `defaultConfig` prop as a part of this PR?